requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Used to install pinned dependencies
 # Useful for dev/test jobs caches
 # Must be kept in sync with setup.py
-torch == 2.2.0 # This is what is installed in CI today
+torch ~= 2.2.0 # This is what is installed in CI today
```

Comment:
is it intentional to prevent 2.3 or would >= be better?

Also FYI we need to update fms-fsdp: 
https://github.com/foundation-model-stack/fms-fsdp/blob/main/requirements.txt

and probably fms-extras:
https://github.com/foundation-model-stack/fms-extras/blob/main/requirements.txt